Database/MongoDB/Connection.hs

```diff
@@ -172,6 +172,13 @@ openReplicaSetSRV hostname = do
 
 openReplicaSetSRV' :: HostName -> IO ReplicaSet 
 -- ^ Open /secure/ connections (on demand) to servers in a replica set. The seedlist and replica set name is fetched from the SRV and TXT DNS records for the given hostname. The value of 'globalConnectTimeout' at the time of this call is the timeout used for future member connect attempts. To use your own value call 'openReplicaSetSRV'''' instead.
+-- The preferred connection method for cloud MongoDB providers. 
+-- ==== __Example__
```

Comment:
Haddock requires special markers if you are giving a code example: https://haskell-haddock.readthedocs.io/en/latest/markup.html#code-blocks

You can also check what the formatted text looks like if you run
```
$ cabal haddock
```
It will tell you where it put generated docs.